src/Filament/Form/Fields/Concerns/CanAutoFillWhenOnlyOneOption.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Statikbe\FilamentFlexibleContentBlocks\Filament\Form\Fields\Concerns;
+
+trait CanAutoFillWhenOnlyOneOption
```

Comment:
@bverbist Dit zouden we eventueel ook kunnen zien als een enum met functies op de enum die nu in de trait staan.